nmigen_cocotb.py

```diff
@@ -3,12 +3,14 @@
 from nmigen import Fragment
 from nmigen.back import verilog
 from nmigen.cli import main_parser, main_runner
+from nmigen.build.plat import TemplatedPlatform
```

Comment:
Not needed. Clean the diff (newlines) and rebase if it's possible.

Reply:
done